showcase-quarkus-eventsourcing/src/main/resources/db/command/common/V1.0.0__AxonOnMicroprofileTryoutEventsourcing.sql

```diff
@@ -10,7 +10,7 @@ CREATE TABLE "axon_on_microprofile_tryout"."domainevententry" (
 	PAYLOADTYPE VARCHAR(255) NOT NULL, 
 	SEQUENCENUMBER BIGINT NOT NULL, 
 	TIMESTAMP VARCHAR(255) NOT NULL,  
-	TYPE VARCHAR(255) NOT NULL, 
```

Comment:
This file shouldn't be changed. The change should be done within a new file with a new database version using ALTER TABLE.